torch/_dynamo/variables/torch.py

```diff
@@ -320,6 +320,9 @@ def get_state_from_generator():
             assert isinstance(args[0], TensorVariable)
 
             if config.fake_tensor_propagation:
+                unimplemented(
+                    "TODO: make torch.random.set_rng_state work with fake tensor"
```

Comment:
NIt: `with fake tensor in aot autograd`. That's where it fails right ?

Reply:
Yes, you are right. Let me update the comment.